core/templates/newspapers.html

```diff
@@ -25,6 +25,7 @@
         <thead>
         <tr>
             <th class="headerSortDown left_no_border">State</th>
```

Comment:
How about frying state entirely and just using "Location"?  That way if we do offer a setting, the word "Location" still works.  And if we bother with a setting, we could have a setting for the entire location format - just state?  Sure!  City and county?  No problem!  Etc.